views/src/main/java/io/micronaut/views/thymeleaf/ThymeleafViewsRenderer.java

```diff
@@ -88,10 +87,11 @@ public ThymeleafViewsRenderer(AbstractConfigurableTemplateResolver templateResol
 
     @Override
     @Nonnull
-    public Writable render(@Nonnull String viewName, @Nullable Object data) {
+    public Writable render(@Nonnull String viewName, @Nullable Object data,
```

Comment:
The original render method should be left as is. You could create a common protected method where you pass in the context

Reply:
Could you please explain this. Do I need to create an abstract implementation of ViewsRenderer interface?

Reply:
@gasches The problem is that if someone at any point was manually using the `ThymeleafViewsRenderer` and calling the original method, they would get an UnsupportedOperationException now. Previously that would not happen, so this is a breaking change. The original method needs to work as it did before

Reply:
Ok. Left original implementation as it is and create new one with request parameter and WebContext.